src/ui/component/publishForm/view.jsx

```diff
@@ -346,297 +347,308 @@ class PublishForm extends React.PureComponent<Props> {
     const shortUri = buildURI({ contentName: name });
 
     return (
-      <Form onSubmit={this.handlePublish}>
```

Comment:
I just added the `React.Fragment` wrapper and the unsupported message for web here.